torchtune/datasets/_chat.py

```diff
@@ -100,7 +101,7 @@ def _prepare_sample(self, sample: Mapping[str, Any]) -> Tuple[List[int], List[in
         labels = list(np.where(mask, CROSS_ENTROPY_IGNORE_IDX, tokens))
         assert len(tokens) == len(labels)
 
-        return tokens, labels
+        return {"tokens": tokens, "labels": labels}
```

Comment:
Can't believe we haven't been doing this all along :)

Reply:
dictionaries are simpler, more generalizable :)